Validation/DTRecHits/test/DTValidation_RelVal_fromRECO_local_cfg.py

```diff
@@ -36,8 +36,8 @@
 ## Uncertainty DB
 if uncertDB != "" : 
     process.GlobalTag.toGet = cms.VPSet(
-        cms.PSet(record = cms.string("DTRecoUncertaintiesRcd"),
-                 tag = cms.string("DTRecoUncertainties_test"),
+        cms.PSet(record = cms.string("DTRecoConditionsUncertRcd"),
+                 tag = cms.string("UncertDB"),
                  connect = cms.untracked.string("sqlite_file:"+uncertDB))
```

Comment:
I'm not sure what's happening here, there is a local sqlite file that's read here? `DTRecoConditionsUncertRcd` is still kept in the GT, right? So why not just use what's in the GT instead of a specification here?

Reply:
This snippet (which is for local tests, i.e. for running manually for testing purposes) allows also, optionally, to take alternate configurations from a .db file (we sometime need to test stuff before we put it them in GTs...)